test/GENoutputTest.html

```diff
@@ -75,7 +75,6 @@ <h3>GEN parameters in SPOT:</h3>
     </script>
```

Comment:
This file is running GENoutputTest() twice, once when GENoutputTest.js is loaded and then again in line 86.